exchange.lua

```diff
@@ -1018,17 +1016,17 @@ end
 
 
 -- On success, returns true and a list of inbox entries.
--- On failure, returns false and an error message.
+-- On failure, an error message.
```

Comment:
What's the reason for this change? This makes it seem like the function might return only an error message on failure (and I do have other functions that do something like that).